mgr/config/v0.2.3-rhel7-root6.24.06-64b.config

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env csh
+
+# this is in place of buggy `module clear` (module VERSION=3.2.10 DATE=2012-12-21)
+unsetenv _LMFILES_*
+module purge
+module unuse /cvmfs/star.sdcc.bnl.gov/star-spack/spack/share/spack/modules/linux-rhel7-x86/
+module use   /cvmfs/star.sdcc.bnl.gov/star-spack/spack/share/spack/modules/linux-rhel7-x86_64/
+module load star-env-0.2.3-root-6.24.06
+module load star-table-0.1.0-root-6.24.06
```

Comment:
Do we really need to load star-table here? Does the environment work without it?
star-table module loads the libraries built from this code https://github.com/star-bnl/star-table

Reply:
No, we don't need to load the table module... it is already provided by the star environment module.  Did a quick check... codes which depend on libTable compile w/out problems.